config/src/config/quorum_store_config.rs

```diff
@@ -29,14 +29,14 @@ impl Default for QuorumStoreBackPressureConfig {
         QuorumStoreBackPressureConfig {
             // QS will be backpressured if the remaining total txns is more than this number
             // Roughly, target TPS * commit latency seconds
-            backlog_txn_limit_count: 12_000,
+            backlog_txn_limit_count: 36_000,
             // QS will create batches at the max rate until this number is reached
-            backlog_per_validator_batch_limit_count: 4,
+            backlog_per_validator_batch_limit_count: 20,
```

Comment:
This means, with 130 validators, proof count backpressure triggers only when the number of proofs in the pipeline is 130 * 20 = 2600 proofs. So, it basically never triggers.

Reply:
Yeah, I think txn count backpressure should be enough